### PR TITLE
修复 dotfile 在 make 时因 ruby 环境选择错误导致的安装失败

### PR DESCRIPTION
**Pull request by undancer**

### CODE REVIEW

File: system/.path.macos [hunk 1]
```diff
@@ -1,2 +1,2 @@
 prepend-path "/usr/local/opt/coreutils/libexec/gnubin"
-prepend-path "/usr/local/opt/ruby/bin"
+prepend-path "/usr/local/opt/ruby/bin"
```
Comment (webpro) [inline system/.path.macos]: Please leave the newline here, no whitespace changes needed.

Comment (undancer) [reply system/.path.macos]: OK. Fix it.

**Comment by webpro** [comment]
Thanks, @undancer! This is probably my first merged PR ever in Chinese!

**Comment by undancer** [comment]
> Thanks, @undancer! This is probably my first merged PR ever in Chinese!

抱歉，我怕我用英文会表达不清。

**Pull request merged**